Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -617,7 +617,7 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Pull deltas from Collection SG", "
     constexpr size_t kDocBufSize = 60;
 
     // connection closing from SGW for 1000 docs => reduced to 50
-    constexpr int kNumDocs = 50, kNumProps = 50;
+    constexpr int kNumDocs = 1000, kNumProps = 50;
```

Comment:
The original test with Walrus had both **kNumDocs = 1000, kNumProps = 1000;**. Could you please change it back to that and test as well?